packages/mc-http-server/server.js

```diff
@@ -178,6 +178,7 @@ const launchServer = async options => {
     });
     prometheusMetricsServer = await createPrometheusMetricsServer({
       port: prometheusMetricsServerPort,
+      detectKubernetes: true,
```

Comment:
Let's see. Basically the new version of promster is better as it does work a bit like lightship. In that it won't start locally if this flag is passed.

TIL: passing `undefined` to `listen` seems to silently not have a server start 🤷‍♂. At least that's what lightship does.